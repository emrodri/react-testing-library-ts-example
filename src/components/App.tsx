import React from 'react'

const App = (): JSX.Element => (<div><h2>React TS skeleton</h2></div>);

export default App;