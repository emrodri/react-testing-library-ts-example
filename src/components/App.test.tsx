import { cleanup, render } from '@testing-library/react';

import App from './App';
import React from 'react';

afterEach(cleanup);

describe('App', () => {
    it('renders properly', () => {
        const { asFragment } = render(<App />);
        expect(asFragment()).toMatchSnapshot()
    })
})