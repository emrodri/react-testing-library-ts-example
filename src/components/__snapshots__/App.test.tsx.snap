// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders properly 1`] = `
<DocumentFragment>
  <div>
    <h2>
      React TS skeleton
    </h2>
  </div>
</DocumentFragment>
`;
